std/utf.d

```diff
@@ -2681,6 +2722,9 @@ if (isSomeChar!C)
 /* =================== Validation ======================= */
 
 /++
+    $(RED Scheduled for deprecation. This is going to be deprecated in favor of
+          $(LREF isValidUTF) in 2.075.)
```

Comment:
No. It's not. The idea is that the release that introduces isValidUTF should not also deprecate validate, because otherwise, you can't have both the latest release and master build without deprecation warnings. However, I do need to up the version by one, since 2.075 is the release that would introduce isValidUTF if it were merged now, making 2.076 the release that should deprecate validate.